react-components/src/data-providers/core-dm-provider/check3dConnectedEquipmentQuery.ts

```diff
@@ -153,3 +124,89 @@ function getObject3dRelation(visualizableTableName: string): QueryTableExpressio
     }
   };
 }
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Remove?

Reply:
Removing it would require us to add return type to the function signature explicitly. However, the object returned here has a very complex type, which again allows us to do some valuable type inference on the return type of `fetchNodesAndEdges`, which is the function that will eventually consume this query object. Writing the full type here is immensely impractical, and narrowing to a simpler type will disallow useful type inference later. I want to keep this comment